integration-test/src/main/java/com/sequenceiq/it/cloudbreak/util/ErrorLogMessageProvider.java

```diff
@@ -22,6 +23,13 @@
 
     private static final ObjectWriter OBJECT_WRITER = new ObjectMapper().writerWithDefaultPrettyPrinter();
 
+    public Throwable getException(Map<String, Exception> exceptionsDuringTest) {
+        return exceptionsDuringTest.values().stream()
+                .filter(exception -> exception.getCause() instanceof TestFailException)
```

Comment:
This will only show the first exception that we encounter in the test, is that correct?
Or we are specifically looking for this type of exceptions among them?
If so this method should be rather, findFirstTestFailException.